### PR TITLE
Migrate travis ci to Github actions.

### PR DESCRIPTION
**Pull request by KaylaNguyen**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      env:
+        GO111MODULE: on
+      run: |
+        go get .
+    - name: Test
+      shell: bash
+      run: |
+        go version
+        go test -v google.golang.org/appengine/...
+        go test -v -race google.golang.org/appengine/...
```
Comment (matthewrobertson) [inline .github/workflows/ci.yml:32]: I am not that familiar with go, why are you running the same tests twice?

**Review by matthewrobertson — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ '1.11.x']
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
```
[the remark this answers is not in the document]
Comment (KaylaNguyen) [reply .github/workflows/ci.yml:19]: Ideally I want to run the tests against at least one other go version, but I'm not sure which one yet and open to suggestions.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (KaylaNguyen) [reply .github/workflows/ci.yml:32]: This test command I copied over from old travis test. Fair point, I think it's safe to just run it once with data race detection on.

File: .github/workflows/ci.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (joelgerard) [reply .github/workflows/ci.yml:19]: I would add 1.11, 1.12, 1.13, 1.14, 1.15 and wait for bergerl to finish 1.16 first.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install
+      env:
+        GO111MODULE: on
```
Comment (joelgerard) [inline .github/workflows/ci.yml:24]: IIUC we need to support gopath users going forward. I assume this means that turning modules OFF. I would check in with DevRel on the status of all this. I believe this was the purpose of the sh files below.

File: .github/workflows/ci.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (KaylaNguyen) [reply .github/workflows/ci.yml:19]: Done. Thanks!

File: .github/workflows/ci.yml (hunk 3 above)
Comment (KaylaNguyen) [reply .github/workflows/ci.yml:24]: I checked with devrel and supporting gomod is the way to go, though gopath is still used by some users due to tech debt. I added tests for both.

**Pull request merged**